hw/ip/otbn/data/otbn.hjson

```diff
@@ -201,6 +201,32 @@
       package: "keymgr_pkg"
     },
   ],
+  features: [
+    {
+      name: "OTBN.ISA"
+      desc: '''
+        OTBN implements a custom ISA based on RV32I. It is split in base (32-bit) and bignum
+        (256-bit) parts. There is a seperate register file for each.
+      ''',
+      name: "OTBN.RND"
+      desc: '''
+        OTBN has a connection to the EDN allowing it provide 256-bits of of entropy to OTBN.
```

Comment:
nit: 
it provide -> it to provide
of of -> of

Reply:
I agree, and also this might need rephrasing a bit. At my initial pass, I understood that OTBN was doing the providing! Maybe "allowing it to receive"?